### ISSUE TITLE
HelloWorld documentation for Pkg doesn't work

### ISSUE THREAD

**Issue by crlaugh** [JuliaLang/Pkg.jl#825]
The HelloWorld example provided at https://docs.julialang.org/en/v1/stdlib/Pkg/#Creating-your-own-packages-1 does not work when followed strictly. The fix provided at https://github.com/JuliaLang/julia/issues/27359 addresses the problem, but this involves adding a dependency, which is only described in the following section (i.e., https://docs.julialang.org/en/v1/stdlib/Pkg/#Adding-dependencies-to-the-project-1). I would try to correct it and submit a PR, but there is something of a philosophical question on how the section should be organized if a dependency is required to execute the MWE; I am therefore hoping that someone closer to this documentation can make the required change.

**Comment by 00vareladavid** [comment]
I'm not sure I understand what the problem is with the HelloWorld example. What dependency are you referring to?

```
               _
   _       _ _(_)_     |  Documentation: https://docs.julialang.org
  (_)     | (_) (_)    |
   _ _   _| |_  __ _   |  Type "?" for help, "]?" for Pkg help.
  | | | | | | |/ _` |  |
  | | |_| | | | (_| |  |  Version 1.0.1 (2018-09-29)
 _/ |\__'_|_|_|\__'_|  |  Official https://julialang.org/ release
|__/                   |

(v1.0) pkg> generate HelloWorld
Generating project HelloWorld:
    HelloWorld/Project.toml
    HelloWorld/src/HelloWorld.jl

(v1.0) pkg> activate HelloWorld

julia> import HelloWorld
[ Info: Precompiling HelloWorld [2e5dc23a-cf31-11e8-1abe-494f67f12647]

julia> HelloWorld.greet()
Hello World!
```

**Comment by crlaugh** [comment]
Thanks for the response.  I think that I mixed up different portions of this code, so that I entered

`]generate HelloWorld
]cd HelloWorld
]activate HelloWorld`

rather than `activate .` as you described.  Following your example works perfectly. 

Now that I see the different examples, it is clearer how this works, but the distinction was sufficiently subtle (and I wasn't sure if I was importing the .jl file or the whole directory) that I messed it up when I was first trying to generate my own package.
